src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,44 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
+
+  constructor(name, health = 100) {
+    this.name = name;
+    this.health = health;
+
+    Animal.alive.push(this);
+  }
+
+  static removeDead() {
+    Animal.alive = Animal.alive.filter((animal) => animal.health > 0);
```

Comment:
The `removeDead` method correctly filters out animals with health less than or equal to zero. Ensure that this method is called appropriately to maintain the integrity of the `alive` list.